### PR TITLE
Fixed typo in views

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**

### CODE REVIEW

File: apps/students/views.py [hunk 1]
```diff
@@ -93,37 +93,37 @@ def attendence(request):
...
         for stu_id in stu_array:
-            stu_att = StudentAttendence.objects.get(student__id=stu_id, cal_date=today_date)
+            stu_att = StudentAttendance.objects.get(student__id=stu_id, cal_date=today_date)
             stu_att.present = True
             stu_att.save()
 
-        messages.success(request, 'Attendence marked successfully!')
-        return redirect('students:attendence')
+        messages.success(request, 'Attendance marked successfully!')
+        return redirect('students:attendance')
```
Comment (garg3133) [inline apps/students/views.py:108]: First line is fine as it is just a text. For the second line, make sure you've changed the name in `urls.py` file also.

File: home/views.py [hunk 2]
```diff
@@ -79,24 +79,24 @@ def dashboard(request):
         context['cw_hw'] = cw_hw
 
         # Students Attendance
-        student_attendence = StudentAttendence.objects.filter(cal_date=calendar, present=True).order_by('student__school_class')
-        context['student_attendence'] = student_attendence
+        student_attendance = StudentAttendance.objects.filter(cal_date=calendar, present=True).order_by('student__school_class')
```
Comment (garg3133) [inline home/views.py:82]: Changing variable name is fine but don't change `StudentAttendence` as it is an import.

File: home/views.py [hunk 3]
```diff
@@ -79,24 +79,24 @@ def dashboard(request):
...
+        context['student_attendance'] = student_attendance
```
Comment (garg3133) [inline home/views.py:83]: While changing context variable name, make sure you also change it anywhere it is being used in corresponding HTML file (`templates/home/dashboard.html` here)

**Comment by garg3133** [comment]
If you have any other doubts, feel free to ping me here :)

File: apps/students/views.py [hunk 4]
```diff
@@ -81,7 +81,7 @@ def attendence(request):
                     stu_attendance.save()
     else:
         context['no_class_today'] = True
-        return render(request, 'students/attendence.html', context)
+        return render(request, 'students/attendance.html', context)
```
Comment (garg3133) [inline apps/students/views.py:84]: You need to change the name of the HTML files also present in `templates/<path-mentioned-above>` folder.

File: apps/volunteers/views.py [hunk 5]
```diff
@@ -76,14 +76,14 @@ def attendence(request):
...
-        return render(request, 'volunteers/attendence.html', context)
+        return render(request, 'volunteers/attendance.html', context)
 
     if request.method == 'POST':
         vol_array = request.POST.getlist('volunteered')
         extra_vol_array = request.POST.getlist('extra-vol')
 
         # Mark everyone's absent
-        vol_att_today = VolunteerAttendence.objects.filter(cal_date=today_cal)
+        vol_att_today = VolunteerAttendance.objects.filter(cal_date=today_cal)
```
Comment (garg3133) [inline apps/volunteers/views.py:86]: You forgot to skip this.

File: home/views.py [hunk 6]
```diff
@@ -12,7 +12,7 @@
 # local Django
 from accounts.models import Profile
 from apps.students.models import Student, StudentAttendence, StudentSchedule
-from apps.volunteers.models import Volunteer, VolunteerAttendence, VolunteerSchedule
+from apps.volunteers.models import Volunteer, VolunteerAttendance, VolunteerSchedule
```
Comment (garg3133) [inline home/views.py:15]: Here also, skip this.

File: templates/home/dashboard.html [hunk 7]
```diff
@@ -160,7 +160,7 @@ <h4 class="text-center mt-3">Students Present</h4>
                             </tr class="text-center">
                         </thead>
                         <tbody>
-                            {% for stu_att in student_attendence %}
+                            {% for stu_att in student_attendance %}
```
Comment (garg3133) [inline templates/home/dashboard.html:163]: You forgot to replace volunteer_attendence here.

**Comment by garg3133** [comment]
Also, change the typo everywhere in the HTML files as typos there are all independent after this PR.

**Comment by 96RadhikaJadhav** [comment]
@garg3133  Done. Please review and let me know if any other changes required.

**Comment by garg3133** [comment]
Two last changes:
- Change the name of `templates/volunteers/attendence.html` file also.
- Fix `attendence` typo in all HTML files (`students/attendance.html`, `volunteers/attendance.html` and `sidebar.html`).

**Comment by 96RadhikaJadhav** [comment]
@garg3133  Done. please review and let me know if any other changes required.

**Comment by garg3133** [comment]
Changes look great now! Merging them.
Thanks @96RadhikaJadhav :)

**Pull request merged**